powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/lib/dataadapter/api/DataAdapter.java

```diff
@@ -34,55 +35,50 @@
     /**
      * Authenticate user using provided credentials.
      *
-     * @param authenticationRequest Authentication request.
-     * @return Authentication response.
-     * @throws MethodArgumentNotValidException Thrown when input validation fails.
+     * @param username Username for user authentication.
+     * @param password Password for user authentication.
      * @throws AuthenticationFailedException Thrown when authentication fails.
      */
-    AuthenticationResponse authenticateUser(AuthenticationRequest authenticationRequest) throws MethodArgumentNotValidException, AuthenticationFailedException;
+    void authenticateUser(String username, String password) throws AuthenticationFailedException;
```

Comment:
We should return something from this method, for example an object with user information (at least, user ID has to be returned). It feels weird that we only let the method go and let it fail on authentication failure...